### DISCUSSION TITLE
Is the project dead?

### DISCUSSION THREAD

**Discussion by albertoloscerritos** [RafaelGB/obsidian-db-folder#954] [General]
Just want to know

**Comment by RafaelGB** [comment]
Hello @albertoloscerritos , currently I am working on another plugin. Since Obsidian itself is working on properties, views and metadata editors, I want to wait until to see how that native update ends.

Once they have finished it, we can   resume the work with this project

**Reply by albertoloscerritos** [reply]
Thanks for your answer. Great to hear that. Hope to see what's coming for this amazing plugin. I think this plugin can result in even better notion-like databases than the ones in notion. But there is still some work to be done.
Have a nice day :)
Thanks
